server/imports/github-methods.js

```diff
@@ -133,3 +134,42 @@ export function putPageContent (userId, fullName, pageName, cb) {
   }, cb)
 
 }
+
+export function tagGhPages (userId, fullName, cb) {
+  let github = githubInterface(userId)
+
+  async.waterfall([
+    // Get current project version
+    Meteor.bindEnvironment(function (cb) {
+      github.getFileContents(fullName, 'package.json', cb)
+    }),
+    // Increment version
+    Meteor.bindEnvironment(function (res, cb) {
+      let pkg = JSON.parse(base64.decode(res.content))
+      pkg.version = incrementPatch(pkg.version)
+      cb(null, pkg, res.sha)
+    }),
```

Comment:
Should we consider storing this information locally rather than getting the package version and sha from Github when we want to release?  It seems like if this *isn't* what we have locally, we would want some sort of message to tell us so, rather than just publishing whatever happens to be in the repo.  If somebody else (e.g. TABLFLIP) makes changes to the repo outside the platform client, this just allows them to go ahead and publish regardless, even if the current data schema is out of date.